corporate/views/portico.py

```diff
@@ -359,3 +361,39 @@ def communities_view(request: HttpRequest) -> HttpResponse:
             "org_types": org_types,
         },
     )
+
+
+@zulip_login_required
+def invoices_page(request: HttpRequest) -> Union[HttpResponse, HttpResponseRedirect]:
+    user = request.user
+    assert user.is_authenticated
+
+    if not user.has_billing_access:
+        context: Dict[str, Any] = {
+            "admin_access": user.has_billing_access,
+            "has_active_plan": False,
+            "org_name": user.realm.name,
+            "billing_base_url": "",
+        }
+
+        return render(request, "corporate/billing/billing.html", context=context)
```

Comment:
What's the thinking for this code path? I feel like it might be better to serve an error or redirect than rendering a different template that is likely not going to be tested for accepting the parameters from this alternative code path.

Reply:
Okay, added a redirect to billing page which shows this message.
<img width="1557" alt="Screenshot 2024-02-02 at 9 00 06 AM" src="https://github.com/zulip/zulip/assets/25124304/4b380b92-efb5-413c-b73f-dd6261bdbb67">